_data/meltano/utilities/evidence/meltanolabs.yml

```diff
@@ -0,0 +1,254 @@
+commands:
+  describe:
```

Comment:
@kgpayne I'm not super familiar with the workflow of deploying this but my understanding is that you'd need to rebuild/deploy after your data refreshes. For this case, would we want a command for `build` so someone could have a schedule like `tap-github target-duckdb evidence:build` to refresh their reports once the data is updated. A command would be useful if someone would want to run it in a schedule.

Reply:
@pnadolny13 as evidence doesn't have a CLI as such, the extension actually behaves as one. I.e.

- `meltano invoke evidence build` runs `npm --prefix $EVIDENCE_HOME run build` under the hood.
- `meltano invoke evidence dev` runs `npm --prefix $EVIDENCE_HOME run dev` under the hood.
- `meltano invoke evidence npm <args>` runs `npm --prefix $EVIDENCE_HOME <args>` for running arbitrary npm commands (like audit or update).

On that basis, I see it makes sense to have a standard command for `build` but maybe not for `dev` 🙂 WDYT?

Reply:
@kgpayne yeah that makes sense to me! Definitely seems good to have a `build` command but maybe not for dev/npm, I'm not really sure 🤷 . I don't know evidence very well so its hard to predict how users will want to use it.

Reply:
Added `build` ✅ Thanks 🙌